src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codepipeline.py

```diff
@@ -482,3 +489,26 @@ def import_required_arns():
             # pylint: disable=no-value-for-parameter
             _output.append(core.Fn.import_value(arn))
         return _output
+
+
+    def add_pipeline_trigger(self, trigger_type, trigger_config):
+        if trigger_type in self._accepted_triggers:
+            trigger_type = self._accepted_triggers[trigger_type]
+        else:
+            LOGGER.error(f"{trigger_type} is not currently supported. Supported values are: {self._accepted_triggers.keys()}")
+            raise Exception(f"{trigger_type} is not currently supported as a pipeline trigger")
```

Comment:
Exiting early, so we don't need the else and reduce the indentation depth.

```suggestion
        if trigger_type not in self._accepted_triggers:
            LOGGER.error(f"{trigger_type} is not currently supported. Supported values are: {self._accepted_triggers.keys()}")
            raise Exception(f"{trigger_type} is not currently supported as a pipeline trigger")
        trigger_type = self._accepted_triggers[trigger_type]
```